docs/cristin.ttl

```diff
@@ -260,39 +286,39 @@
     :isignored   "true" .
 
 :arstall_rapportert
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment "Year that the resource was included in an NVI report." ;
     rdfs:range   xsd:positiveInteger ;
     rdfs:domain  :Result ;
     :isignored   "true" .
 
 :merknadtekst
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment "Free text for super-users to add comments when it comes to NVI approvals." ;
     rdfs:range   xsd:string ;
     rdfs:domain  :Result ;
     :isignored   "true" .
 
 :merknadtekst_godkjenning
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment "Free text fore super-users to add explanations/details on why the publication was approved or rejected." ;
     rdfs:range   xsd:string ;
     rdfs:domain  :Result ;
     :isignored   "true" .
 
 :kildekode
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment """
     The source (provider) the result was imported from, if any.
     Mostly superseded by the list of import references in table VARBEID_KILDE
-    It is a string containing arbitrary codes such as "scopus" "norart" etc.
+    It is a string containing arbitrary codes such as "scopus" "norart", "BIBSYS" etc.
```

Comment:
NB: This, together with kildepostid, is something we have to check more closely, because we also have a list/array VARBEID_KILDE which can include more than source, say when two records from different sources were merged into one Cristin result.

PS: Ah, I see you noted something along these lines on "kildepostid". So yeah, maybe enough to note that one should rather look at VARBEID_KILDE